spec/time_tracker_spec.rb

```diff
@@ -0,0 +1,9 @@
+# Needed to reproduce the issue:
+# https://github.com/KnapsackPro/knapsack_pro-ruby/pull/265
+require 'spec_helper'
+
+describe 'Test Time Tracker' do
+  it do
+    expect(true).to be true
```

Comment:
Is it testing Time Tracker?

Reply:
> Is it testing Time Tracker? 

This spec does not directly do it. It's done by a side effect of running this spec on CI for the knapsack_pro gem:
https://github.com/KnapsackPro/knapsack_pro-ruby/pull/265

> I guess it's only the file itself that's needed to run tests in ruby client circle ci tests?

Yes. I'll rename it.